src/test/lambda/commands/deploySamApplication.test.ts

```diff
@@ -113,21 +113,21 @@ describe('deploySamApplication', async () => {
         getCredentials: async () => testCredentials,
     }
 
-    let channelLogger: FakeChannelLogger
-
     let samDeployWizardResponse: SamDeployWizardResponse | undefined
     const samDeployWizard = async (): Promise<SamDeployWizardResponse | undefined> => {
         return samDeployWizardResponse
     }
 
     let tempToolkitFolder: string
     beforeEach(async () => {
-        channelLogger = new FakeChannelLogger()
-
         tempToolkitFolder = await makeTemporaryToolkitFolder()
         const templatePath = path.join(tempToolkitFolder, 'template.yaml')
         writeFile(templatePath)
 
+        // TODO: is this safe? will add output channel across all tests
+        // we are using this pattern in other tests...
+        ext.outputChannel = vscode.window.createOutputChannel('test channel')
```

Comment:
Agreed, this is very sus

Reply:
It might be best to add any writes to `ext` to the `globalSetup` file, which is what the loggers do. Definitely seems more appealing than going through and adding a param for `ext` to all of our tests that use it.

Reply:
Circling back around to this, we'll probably want a separate PR to address this. Problem is, we can't set `ext` values back to `undefined`.